### DISCUSSION TITLE
Github's API is not returning Status in it's response header

### DISCUSSION THREAD

**Discussion by bhavinmatariya** [community/community#24317] [Apps, API and Webhooks]
<p>Hi, recently I have faced the issue of github’s public API’s are not returning status field in the response header as mentioned in the <a href="https://docs.github.com/en/rest/overview/resources-in-the-rest-api" class="inline-onebox" rel="noopener nofollow ugc">Resources in the REST API - GitHub Docs</a> Doc,<br>
because of this we are blocked with one of the release as build is not going further and returning error<br>
I have found one of the thread <a href="https://github.com/jspm/github/issues/129" class="inline-onebox" rel="noopener nofollow ugc">error while installing jspm packages · Issue #129 · jspm/github · GitHub</a> where we have found out this issue is going because of the public API’s</p>

**Comment by imwiss** [comment, marked as answer]
<p>Hi <a class="mention" href="/bhavinmatariya">@bhavinmatariya</a> :wave: This is a change from our (GitHub’s) side where the <code>status</code> header is no longer being returned as of very recently. The <code>status</code> header was previously included by the rack server we use and the newer version has removed it. The infrastructure team confirmed that we won’t be able to roll this back and restore the header unfortunately.</p>
<p>The workaround is to use the HTTP status itself, rather than the <code>status</code> header. Apologies if this caused downtime for you, this was an unfortunate side effect on the upgrade and not how we would usually announce breaking changes. I’ve noticed that the <code>jspm</code> is implementing a solution for this, which should hopefully unblock you.</p>
<p>As for the documentation, thank you for flagging the inconsistency there. We’ll be updating the docs to reflect the recent changes.</p>
